inventory-management/src/main/java/com/smartnerd/daoImp/DaoImp.java

```diff
@@ -0,0 +1,29 @@
+package com.smartnerd.daoImp;
+
+import javax.transaction.Transactional;
+
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+import org.springframework.stereotype.Repository;
+
+import com.smartnerd.dao.Dao;
+@Repository
+@Transactional
+public class DaoImp implements Dao {
+	private  SessionFactory sessionFactory = null;
+	public  SessionFactory getSessionFactory(){
```

Comment:
spring configuration directly give session factory obj. why you are manually creating?